sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -120,6 +124,9 @@ export default {
       ]);
       // TODO: Move formatted address to a common place in Vue app or API.
       // Adjust the spaces when optional fields are not present.
+      isReadOnly.value =
+        applicationData.applicationStatus !== ApplicationStatus.draft ||
+        props.readOnly;
```

Comment:
I believe that the below expression must have the second part converted to a boolean (using double exclamations), as below.
```
isReadOnly.value =
        applicationData.applicationStatus !== ApplicationStatus.draft ||
        !!props.readOnly;
```

Reply:
I do not see a benefit of converting an object to boolean here as we take the advantage of objects being used in expression. The logic works perfectly with given verbose.

Reply:
What type of object do you expect as a result in "isReadOnly.value"?

Reply:
- This PR changed the Form.IO component readOnly property from string to boolean (I am assuming that was because it reflects better what it really is).
- isReadOnly.value if initialized as boolean, because I assume that it is better expressed as a boolean
- The mentioned expression could return a boolean or a string, so I was just assuming that it would be more accurate to keep the isReadOnly.value always populated with a boolean instead of sometimes as a string, as shown below.

![image](https://user-images.githubusercontent.com/61259237/134379732-5aaa6c8d-510f-4d6a-bbde-5f6f0d879ff9.png)

![image](https://user-images.githubusercontent.com/61259237/134379746-4c444cc6-bdb9-4f60-b221-0d4c24cf6414.png)

Reply:
I got it now. It is getting tricky with the scenario. Will modify accordingly